pkgs/development/python-modules/pyunpack/default.nix

```
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, fetchFromGitHub
+, pytestCheckHook
+, easyprocess
+, entrypoint2
+, patool
+}:
+
+buildPythonPackage rec {
+  pname = "pyunpack";
+  version = "0.3";
```

```suggestion
  version = "0.3";
  format = "setuptools";
```
Would you please add `format`?